services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -98,20 +98,20 @@
 router = APIRouter()
 
 
-async def _check_pipeline_not_running(
+async def _check_pipeline_not_running_or_raise_409(
     comp_tasks_repo: CompTasksRepository, computation: ComputationCreate
 ) -> None:
     pipeline_state = utils.get_pipeline_state_from_task_states(
         await comp_tasks_repo.list_computational_tasks(computation.project_id)
     )
     if utils.is_pipeline_running(pipeline_state):
         raise HTTPException(
-            status_code=status.HTTP_403_FORBIDDEN,
+            status_code=status.HTTP_409_CONFLICT,
             detail=f"Project {computation.project_id} already started, current state is {pipeline_state}",
         )
 
 
-async def _check_pipeline_startable(
+async def _check_pipeline_startable_or_raise(
```

Comment:
really? a check always raises otherwise it's not a check. but if that is clearer.. ok.

Reply:
Yes, perhaps here is unnecessary. I added it in the other because we got the code wrong ... 
In any case we have to change this approach. Instead we should raise custom exceptions and translate them later to status-code using exception handlers ... That is more reliable than what we do now